docs/checks/fixMeReviewCheck.md

```diff
@@ -0,0 +1,21 @@
+# FixMeReviewCheck
+
+#### Description
+The purpose of this check is to flag features that contain the "fixme"/"FIXME" tags with along with a variety of other important tags.
```

Comment:
Could you add a further description of how the supplemental tags and supported values are used?

Reply:
addressed